docs/user_manual/yaydoc_configuration.md

```diff
@@ -0,0 +1,135 @@
+In order to generate and deploy documentation for a registered repository, Yaydoc reads configurations from a YAML file. 
+To get started with Yaydoc the first step is to create a file named `.yaydoc.yml` and specify the required options.The 
+configuration file is divided into four sections. 
+
+## metadata
+Following is a description of config options under the `metadata` section and it’s example usage.
+
+<!-- markdown+ -->
+| Key | Description | Default |
+|-----|-------------|---------|
+| author      | The author of the repository. It is used to construct the copyright text.                | user/organization |
+| projectname | The name of the project. This would be displayed on the generated documentation.         | Name of the repository |
+| version     |	The version of the project. This would be displayed alongside the project name           | Current UTC date |
+| debug       | If true, the logs generated would be a little more verbose. Can be one of true or false. | true |
+| inline_math | Whether inline math should be enabled. This only affects markdown documents.             | false |
+| autoindex   | This section contains various settings to control the behavior of the auto generated index. Use this to customize the starting page while having the benefit of not having to specify a manual index. | – |
+<!-- endmarkdown+ -->
+
+```yaml
+metadata:
+  author: FOSSASIA
+  projectname: Yaydoc
+  version: development
+  debug: true
+  inline_math: true
+  autoindex:
+    include:
+    - README.md
+    toc:
+      show: true
+      heading: Contents
+    subproject:
+      show: true
+      heading: Sub projects
+    apidoc:
+      show: true
+      heading: API Documentation
+      javadoc:
+        show: true
+      swagger:
+        show: true
+    rss:
+      heading: RSS
+      url: <rss_url>
+    twitter:
+      heading: Tweets
+      query: fossasia
+```
+
+## build
+Following is a description of config options under the `build` section and it’s example usage.
+
+<!-- markdown+ -->
+| Key | Description | Default |
+|-----|-------------|---------|
+| theme            | The theme which should be used to build the documentation. The attribute name can be one of the built-in themes or any custom sphinx theme from PyPI. Note that for PyPI themes, you need to specify the distribution name and not the package name. It also provides an attribute options to control specific theme options. | sphinx_fossasia_theme |
+| source           | This is the path which would be scanned for markdown and reST files. Also any static content such as images referenced from embedded html in markdown should be placed under a _static directory inside source. Note that the README would always be included in the starting page irrespective of source from the auto-generated index | docs/ |
+| logo             | The path to an image to be used as logo for the project. The path specified should be relative to the source directory. | – |
```

Comment:
Replace '-'

Reply:
with? `none`?

Reply:
Replace hyphen with hyphen. It's just that the copied text is in incorrect encoding. When you change the encoding of the document to utf-8 and type `-`, It would properly render.